lib/cylc/suite_srv_files_mgr.py

```diff
@@ -540,7 +540,6 @@ def _get_suite_title(self, reg):
             match = re.match('^\s*title\s*=\s*(.*)\s*$', line)
             if match:
                 title = match.groups()[0].strip('"\'')
-                break
```

Comment:
Yes. It was not there before, and I'm not sure if I can break once the match is found ( as if title is matched then there is no need to continuing and matching other lines, but was not sure if that affects anything else so removed)